config/main.py

```diff
@@ -1128,5 +1173,17 @@ def naming_mode_alias():
     set_interface_naming_mode('alias')
 
 
+#
+# 'tx_error_stat_poll_period' subcommand ('config tx_error_stat_poll_period')
+#
+@config.command()
+@click.argument('period', metavar='<period>', required=True)
+def tx_error_stat_poll_period(period):
+    """Set polling period of tx error statistics, 0 for disable, xxx for default"""
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    config_db.set_entry("TX_ERR_CFG", ("GLOBAL_PERIOD"), {"tx_error_check_period": period})
```

Comment:
need to check if "period" is a valid number.

Reply:
solved.